lgc/builder/BuilderRecorder.cpp

```diff
@@ -1383,6 +1385,32 @@ Value *BuilderRecorder::CreateReadGenericInput(Type *resultTy, unsigned location
                 instName);
 }
 
+// =====================================================================================================================
+// Create a read of (part of) a perVertex input value, passed from the previous shader stage.
+//
+// @param resultTy : Type of value to read
+// @param location : Base location (row) of input
+// @param locationOffset : Variable location offset; must be within locationCount
+// @param elemIdx : Vector index
+// @param locationCount : Count of locations taken by the input
+// @param inputInfo : Extra input info (FS interp info)
+// @param vertexIndex : Vertex index
```

Comment:
'Vertex index' is used as the description for both `elemIdx` and `vertexIndex`. What's the difference between the two?

Reply:
"VertexIndex" is for vertex index, For each vertex, it is unique. the range is 0~2, up to three vertices per primitive.
"elemIdx" is for vector component index, the range is 0~3.

Reply:
Thanks for the explanation. Could you add it as a code comment?